doc/api.md

```diff
@@ -1218,6 +1218,7 @@ Config group is represented by a JSON object with the following fields:
 | `ngsiVersion`                  | ✓        | string         |            | optional string value used in mixed mode to switch between **NGSI-v2** and **NGSI-LD** payloads. Possible values are: `v2` or `ld`. The default is `v2`. When not running in mixed mode, this field is ignored.                                                           |
 | `defaultEntityNameConjunction` | ✓        | string         |            | optional string value to set default conjunction string used to compose a default `entity_name` when is not provided at device provisioning time.                                                                                                                         |
 | `autoprovision`                | ✓        | bool           | ✓?         | optional boolean: If `false`, autoprovisioned devices (i.e. devices that are not created with an explicit provision operation but when the first measure arrives) are not allowed in this group. Default (in the case of omitting the field) is `true`.                   |
+| `payloadType`                  | ✓        | string         |            | optional string value used to switch between **IoTAgent**, **NGSI-v2** and **NGSI-LD** measure payloads types. Possible values are: `IoTAgent`, `NGSIv2` or `NGSILD`. The default is `IoTAgent`.                                                                          |
```

Comment:
It seems we are using this in lowercase. In PR https://github.com/telefonicaid/iotagent-json/pull/779 I see:

```
   PAYLOAD_NGSIv2: 'ngsiv2',
   PAYLOAD_NGSILD: 'ngsild',
```

(I guess we are using a tolowercase() in some place)

Maybe it would be better (and closer to actual usage) to describe this in lowercase:

```suggestion
| `payloadType`                  | ✓        | string         |            | optional string value used to switch between **IoTAgent**, **NGSI-v2** and **NGSI-LD** measure payloads types. Possible values are: `iotagent`, `ngsiv2` or `ngsild`. The default is `iotagent`.                                                                          |
```